src/frontends/pytorch/src/op_table.cpp

```diff
@@ -905,6 +906,7 @@ const std::unordered_map<std::string, CreatorFunction> get_supported_ops_fx() {
         {"aten.div.Tensor", op::translate_div_fx},
         {"aten.div.Tensor_mode", op::translate_div_fx},
         {"aten.div_.Tensor", op::translate_div_fx_},
+        {"aten.divide", op::translate_div_fx},
```

Comment:
do you observe this alias when exported to torch.fx?

Reply:
It appears so-> optimum_intel_exports has a fallback mechanism: if exporting from Torch(HF PreTrained Class) to IR fails, it falls back to exporting from Torch to ONNX and then from ONNX to IR. Therefore, if I don't add this alias in get_supported_ops_fx, the export process fails.